test/automation/src/sql/notebook.ts

```diff
@@ -157,6 +172,17 @@ export class Notebook {
 		await this.code.waitForElement(textSelector, result => result?.textContent === text);
 	}
 
+	async isTextCellInEditMode(): Promise<boolean> {
+		// verify that cell is in editmode
```

Comment:
These comments you've added aren't very useful. If you do add comments they should typically be explaining why something is done - for example here you could explain why these classes are used for picking out which cell is in edit mode.